scripts/test/IndirectCommonTests.py

```diff
@@ -9,7 +9,7 @@
 These scripts are used by the ISIS Indirect geometry interfaces such as Indirect Convert to Energy,
 Data Analysis, and Bayes.
 """
-import os
+import posixpath
```

Comment:
`posixpath` is not used directly here and `os` is removed but still used in the test? Is the import of `posixpath` required now?